docs/rfc/rfc-8.adoc

```diff
@@ -0,0 +1,410 @@
+:toc: macro
+
+= RFC 8: Cross-chain Tokenized Threshold BTC
+
+:icons: font
+:numbered:
+toc::[]
+
+== Goal
+
+The goal of this proposal is to enable *efficient* cross-chain tBTC liquidity.
+
+In the Ethereum ecosystem, that means bringing Bitcoin to L2s and sidechains like
+Arbitrum, Optimism, Polygon, Avalanche, and Binance Smart Chain. In the Cosmos
+ecosystem, that means Bitcoin on IBC-enabled zones (chains) like Osmosis and
+Penumbra. And in newer ecosystems, that means chains like Solana, Aptos, and
+Sui.
+
+Efficiency requires minimizing liquidity fragmentation by avoiding the need for
+tBTC wrapper tokens by each chain and bridge. This situation quickly leads to an
+`n*m` problem, where liquidity on `n` chains is divided between `m`
+bridges. Each of these token variants soaks up valuable liquidity that could be
+better utilized elsehwere.
+
+Because tBTC is minted "centrally" on Ethereum L1, we can do better. By
+deploying a canonical token on each chain, we can ensure the supply of tBTC
+remains sacrosanct, while enabling quick, interoperable cross-chain bridges and
+localizing ecosystem risk.
+
+== Overview
+
+To achieve this goal, I propose we deploy a canonical tBTC contract on each
+chain we support. This contract should be flexible enough to:
+
+* Delegate restricted minting authority to a native bridge on the chain, if
+  present.
+* Delegate restricted minting authority to a short list of ecosystem bridges
+* Be paused by any one of `n` guardians, allowing avoidance of contagion in case
+  of a chain- or bridge-specific incident.
+* Be governed by the Threshold Council until we can land on a longer-term
+  cross-chain governance mechanism.
```

Comment:
I would love to see the explicit UX plan in the overview: that initially we're minting normally on ethereum, and then using a separate UI (wormhole) to bridge to L2s, upgrading the bridged tokens into the canonical token that we're deploying

Reply:
This is described in _3.2. Minting on L1_ section if the _Initial Implementation_. Maybe I'll move _Minting on L1_ to be the very first section of the _Initial Implementation_?

Reply:
Done in deb66b5ae043f99f06b9a23f2f6946985df15ff5.